packages/protocol/contracts-0.8/common/ScoreManager.sol

```diff
@@ -0,0 +1,50 @@
+// SPDX-License-Identifier: LGPL-3.0-only
+pragma solidity >=0.8.7 <0.8.20;
+
+import "../../contracts/common/Initializable.sol";
+import "../../contracts/common/interfaces/ICeloVersionedContract.sol";
+import "@openzeppelin/contracts8/access/Ownable.sol";
+
+contract ScoreManager is Initializable, Ownable {
+  mapping(address => uint256) public scores;
```

Comment:
I would actually abstract this, you set scores of addresses, doesn't really matter if it's a group or a validator.

In fact, we have logic to define the score of a group from the score of its members in `Validators.calculateGroupEpochScore`. This function is no longer used because it receives uptimes as parameters, but the logic that I paste could be reused:

```
for (uint256 i = 0; i < uptimes.length; i = i.add(1)) {
      sum = sum.add(FixidityLib.wrap(calculateEpochScore(uptimes[i])));
    }
    return sum.divide(FixidityLib.newFixed(uptimes.length)).unwrap();
 ```
 
 Then we just check if the score manager overwrote that, or the average of its members should be used.

Reply:
I'm ok with having a `getScore(address)` and then just a `scores` mapping. Or having `getValidatorScore` and `getGroupScore` and then 2 mappings. Either way

Reply:
addressed in #11223 commit 29f7acd23152c1b0e2ec722d0d783bcf1e5bbe0b